jsdom/mod.ts

```diff
@@ -0,0 +1,4 @@
+import { jsdom as instance } from "./vendor/jsdom.js";
+import { jsdom } from "./types/jsdom.ts";
+
+export const JSDOM = instance.JSDOM as jsdom.JSDOM;
```

Comment:
Another implementation of this that should also work:

```js
export { jsdom as JSDOM } from "./vendor/jsdom.js";
export { jsdom as JSDOM } from "./types/jsdom.ts";

Reply:
Thanks for the suggestion, @j-f1. Unfortunately, this doesn't work on my machine. When I change it, `JSDOM` can't be imported anymore.

```
$ deno test.ts
Compile file:///Users/marktiedemann/dev/deno_std/jsdom/mod.ts
error: Uncaught SyntaxError: The requested module './mod.ts' does not provide an export named 'JSDOM'
```

Reply:
Well, that’s unexpected. Sorry to send you on a wild goose chase!

Reply:
No worries, @j-f1.

I just tried again, with the same code, and got a different error:

```
$ deno test.ts
Compile file:///Users/marktiedemann/dev/deno_std/jsdom/mod.ts

error TS2300: Duplicate identifier 'JSDOM'.

► file:///Users/marktiedemann/dev/deno_std/jsdom/mod.ts:1:19

1 export { jsdom as JSDOM } from "./vendor/jsdom.js";
                    ~~~~~

error TS2300: Duplicate identifier 'JSDOM'.

► file:///Users/marktiedemann/dev/deno_std/jsdom/mod.ts:2:19

2 export { jsdom as JSDOM } from "./types/jsdom.ts";
                    ~~~~~


Found 2 errors.
```

This seems like a more expected error, still unfortunate.

Reply:
I suspect denoland/deno#2746 will fix this (or at least make it possible).

Reply:
@kitsonk, awesome! As soon as https://github.com/denoland/deno/pull/2746 is released, I will update the types here.